services/common/src/constants/enums.ts

```diff
@@ -155,6 +155,15 @@ export enum PROJECT_STATUS_CODES {
   CHR = "CHR",
 }
 
+export enum MAJOR_MINE_APPLICATION_AND_IRT_STATUS_CODE_CODES {
```

Comment:
This is a really terrible name, I know! And so long! I just couldn't think of anything else and `MAJOR_MINE_APPLICATION_AND_IRT_STATUS_CODES` was already taken. It needed making but I'd be more than happy to see it renamed. 😂